client/my-sites/patterns/components/pattern-preview/style.scss

```diff
@@ -104,6 +104,11 @@
 	}
 }
 
+.pattern-preview--page {
+	break-inside: avoid;
```

Comment:
Hmm, did you set it just for the printing version, or is there another benefit?

Reply:
Firefox sometimes inserts column breaks inside `.page-preview`s without this